EIPS/eip-####.md

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
```suggestion
---
eip: 7688
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.